locals.tf

```diff
@@ -44,6 +44,9 @@ nginx:
 astronomer:
   houston:
     config:
+      email:
```

Comment:
can we make this block conditional similar to how this one is done?
https://github.com/hashicorp/terraform-guides/blob/master/infrastructure-as-code/terraform-0.12-examples/new-template-syntax/rigged_vote.txt
i.e. if var.smtp_uri is "", then omit this block